docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -200,7 +201,7 @@ will be removed in the next major version (8.0).
 [float]
 ==== `hits.total` is omitted in the response if `track_total_hits` is disabled (false)
 
-If `track_total_hits` is set to `false in the search request the search response
+If `track_total_hits` is set to `false` in the search request the search response
 will set `hits.total` to null and the object will not be displayed in the rest
 layer. You can add `rest_total_hits_as_int=true` in the search request parameters
-to get the old format back (`"total": -1`).
\ No newline at end of file
+to get the old format back (`"total": 1`).
```

Comment:
This is for the case where `track_total_hits` is set to false so the old format returns `-1` here and not `1`.